benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/MeasureComposable.kt

```diff
@@ -17,6 +17,8 @@ interface GraphicsContext {
     fun surface(width: Int, height: Int): Surface
 
     suspend fun awaitGPUCompletion()
+
+    fun close() = Unit
```

Comment:
Don't we need to have `close` for macOS/iOS targets? 
BTW, what bad will happen if we don't close GraphicsContext?

Reply:
> what bad will happen if we don't close GraphicsContext?

GPU memory won't be freed, even on GC.

Maybe it was also better to add an additional GC cleaning in Skiko, but generally GPU objects are better to clean when they are not needed.

Reply:
> Don't we need to have close for macOS/iOS targets?

We need actually. I will change the code.

I can free/close the Skia objects, but I can't free the ObjC objects, as Kotlin Native doesn't support that.

Reply:
> I can free/close the Skia objects

Though, there is no point in closing Skia objects if we have bigger issues with the graphic objects. Like `MTLTextureDescriptor` - it allocates a lot of bytes, and we don't free them. And we can't because we allocated them in Kotlin. But it will be freed on GC, and on low memory warning.

The fact that low memory warning takes graphic objects into account is a guess, as macOs/iOS uses a shared system/graphic memory. It wouldn't be the case with Windows/Linux, where memories are often physically separated, and so memory managers with own rules.

I'll keep the current code for simplicity. But for desktop we need to close because we have separated memories

Reply:
>GPU memory won't be freed, even on GC.

But we are quitting from the app just after close.

Reply:
Indeed, here we can skip closing. Then it is just my habit to assume it can be called anywhere, not necessarily in the end of the application.

I can remove it if you think it is better/cleaner without it.

Reply:
Removed to reduce the PR scope